src/resources/commandTemplates/templates.ActionCommand.ts

```diff
@@ -47,7 +47,7 @@ export async function getRuleDescriptions(rule: string): Promise<string> {
   return `${RESOLVED_RULE?.ruleNumber}. ${RESOLVED_RULE?.shortDesc ?? 'Unknown rule'}`;
 }
 
-const durations = {
+export const durations = {
```

Comment:
Maybe we could also move these so they aren't in action command if we're using them more generally?